### ISSUE TITLE
Genre metadata tags not displaying across different players

### ISSUE THREAD

**Issue by m-shustack** [gpac/gpac#2172]
I'm using MP4Box to tag some AAC audio files (using the **-itags option**) and I'm having difficulty getting the Genre metadata to display in the players that I've tested the files in.

The other metadata fields are showing up across all players, but Genre does not show in iTunes, Windows Media Player, VLC, or Audacity. Editing the internal "tag" of the genre from "©gnr" to "©gen" allows the Genre to show in Windows Media Player, VLC, and Audacity, but it still does not display in iTunes. This change is a binary edit, and I haven't found a way to do this in MP4Box.

Is there another way that we need to add Genre metadata so it shows consistently across all players?

**Environment:**

Windows64
GPAC version - latest nightly build: version 2.1-DEV-rev74-gfa127347-master

**Comment by jeanlf** [comment]
You're correct, the user-defined genre was not using the right 4CC, now patched.

On OSX, both built-in genre ('gen' tag) and user ones('@gen' tag) show in Music app.
I don't have iTunes at hand, but given that genre tagging now works properly on OSX I doubt that we can do anything about this.
